packages/ui-utils/src/truncateWithEllipsis.tsx

```diff
@@ -0,0 +1,9 @@
+export const truncateWithEllipsis: (
+  inputText: string,
+  maxLength: number
+) => string = (inputText, maxLength) => {
+  if (inputText.length > maxLength) {
+    return inputText.slice(0, maxLength) + '...';
+  }
+  return inputText;
+};
```

Comment:
Is there a specific reason to make this util? Please use [css/tailwindcss ellipsis](https://tailwindcss.com/docs/text-overflow#ellipsis) where possible.